packages/vue-query/src/__tests__/vueQueryPlugin.test.ts

```diff
@@ -296,6 +296,8 @@ describe('VueQueryPlugin', () => {
     })
 
     test('should delay useQueries subscription and not call fetcher if data is not stale', async () => {
+      const consoleMock = jest.spyOn(console, 'error')
+      consoleMock.mockImplementation(() => undefined)
```

Comment:
why do we need a mock here? it doesn't look like the test would log anything on purpose?

Reply:
There was a console error logged by a vue function, i think I missed it while I updated with v5, will fix and push. Thanks.